floyd/client/files.py

```diff
@@ -7,6 +7,9 @@
 from pathlib2 import PurePath
 from shutil import rmtree
 
+from scandir import scandir
```

Comment:
scandir is included in python since 3.5, can you change this to the following?

```
# Use the built-in version of scandir/walk if possible, otherwise
# use the scandir module version
try:
    from os import scandir, walk
except ImportError:
    from scandir import scandir, walk
```

Reply:
see https://pypi.org/project/scandir/#description

Reply:
Yep! Thanks for the reference.